client/dashboard/sites/settings-transfer-site/confirm-new-owner-form.tsx

```diff
@@ -29,17 +31,37 @@ const form = {
 
 export function ConfirmNewOwnerForm( {
 	siteSlug,
+	newOwnerEmail,
 	handleSubmit,
 }: {
 	siteSlug: string;
-	handleSubmit: ( event: React.FormEvent ) => void;
+	newOwnerEmail: string;
+	handleSubmit: ( data: ConfirmNewOwnerFormData ) => void;
 } ) {
 	const [ formData, setFormData ] = useState( {
-		email: '',
+		email: newOwnerEmail,
 	} );
 
+	const mutation = useMutation( siteOwnerTransferEligibilityCheckMutation( siteSlug ) );
+
 	const isSaveDisabled = ! isItemValid( formData, fields, form );
 
+	const onSubmit = ( event: React.FormEvent ) => {
```

Comment:
Pedantic, but it seems you're using `handle{something}` instead of `on{event}` for locally defined functions in the other files. Do we have a standard? I typically use `handle` for local and `on` for component props.

Reply:
Ok